src/components/AddTask.js

```diff
@@ -70,7 +71,7 @@ const AddTask = () => {
     const addSubtask = (subtask) => {
         console.log(subtasks);
         if(!subtasks.includes(subtask)){
-            setSubtasks([...subtasks, subtask]);
+            setSubtasks([...subtasks, {subtask, editOpen: false}]);
```

Comment:
If you write it that way: `{subtask, editOpen: false}`, it is a shorthand and creates an object like this:
```js
{subtask: <subtask-name>, editOpen: false}
```

subtask as a field is of a subtask can be confusing.